src/games/strategy/triplea/image/UnitImageFactory.java

```diff
@@ -146,9 +146,6 @@ private void clearImageCache() {
     // URL uses '/' not '\'
     final String fileName = FILE_NAME_BASE + id.getName() + "/" + baseImageName + ".png";
     final URL url = resourceLoader.getResource(fileName);
-    if (url == null) {
-      ClientLogger.logError("MISSING IMAGE (this unit or image will be invisible): " + baseImageName + "  looking in: " + fileName);
-    }
     return Optional.ofNullable(url);
```

Comment:
The code attempts to find image files in a bit of a shot-gun approach, and tries many paths which are not expected to be there. So this means we log an error quite frequently, whenever all unit images are being displayed. This means some paths of entry to this code point are happy with missing images, and others are not, so the logging needs to move downstream.